airbyte-webapp/src/utils/numberHelper.tsx

```diff
@@ -0,0 +1,15 @@
+import { FormattedMessage } from "react-intl";
+
+export const formatBytes = (bytes?: number) => {
```

Comment:
Since we have this now properly extracted, we should add some unit tests for this file.